doc/conf.py

```diff
@@ -22,9 +20,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = setup_args['version']
+version = "1.7"
 # The full version, including alpha/beta/rc tags.
-release = setup_args['version']
+release = "1.7dev7"
```

Comment:
I've had to switch to python3 to build it, which does not allow imports like this. I'm no even quite sure how it's currently working since the top level has no ``__init__.py``